crates/hc_sandbox/src/calls.rs

```diff
@@ -147,9 +147,6 @@ pub struct RegisterDna {
     #[arg(long)]
     /// Properties to override when installing this DNA
     pub properties: Option<PathBuf>,
-    #[arg(long)]
-    /// Origin time to override when installing this DNA
-    pub origin_time: Option<Timestamp>,
```

Comment:
There are going to be a bunch of invalid manifests out there. I wonder if this might be a good time to have `serde` reject unknown fields to clean that up. No point having people trying to maintain those fields.

That's something I've done in a few places to help catch spelling errors etc. 

Not on this PR, but what do you think?

Reply:
I don't think I personally care at all about cruft in people's configs, but also totally happy for this to happen : ) https://github.com/holochain/holochain/issues/4776

Reply:
Haha, understood :) I'm happy to take this on because I will care when I have to answer the same question many times :D